README.md

```diff
@@ -1,31 +1,49 @@
+The information is passed to the debugger as an array of individual messages (logs), which you need to create in the script.
 
-The log data presentation and appearance are easy to change with some basic JavaScript and CSS.
+The information is displayed in a pop-up window and/or in the login screen during an authentication journey.
 
-> In addition, by default, the logs will be outputted with the `logger.error(String log)` method. This can be disabled with the `noLoggerError: true` option.
+> You will need to allow pop-ups for your AM origin.
 
-Optionally, you can enable control of this this functionality with a URL query string parameter `&debug=true|false`. It can be enabled by providing the `useDebugParameter` option
+Information displayed in the pop-up window will be continuously updated by instances of the scripted debugger. Thus, as long as the pop-up window stays open, it can serve as a log tailing tool.
 
-## How it works
+Information displayed in the login screen is provided by a single instance of the scripted debugger—the one that is being currently executed.
 
-The [src/scripted-debugger.js](src/scripted-debugger.js) code contains a function and an example of a caller. See comments in the function code for the supported options.
+Both display options are optional and work independently from each other.
+
+The data presentation can be changed by including HTML in individual logs. It is also possible to change the overall appearance with some basic JavaScript and CSS modifications made in the debugger function code.
+
+> In addition, by default, the logs will be outputted with the `logger.error(String log)` method. This can be disabled.
+
+Optionally, you can enable control of this functionality with a URL query string parameter `&debug=true|false` added to the authentication journey URL.
+
+See [docs > showLogs](docs/READMe.md#showlogs) for the supported options.
 
 ## How to make it work
 
-* Copy the code from [src/scripted-debugger.js](src/scripted-debugger.js) and paste it at the bottom of your scripted decision node debugger script.
+* Copy the code from [src/scripted-debugger.js](src/scripted-debugger.js) and paste it at the bottom of your scripted decision.
+
+    > The debugger function itself, as any function statement, can be placed at and called from any place in your script.
 * In your scripted decision node script, create an array of log messages or choose a single message to be displayed in the browser.
-* Using the provided example, call the `showLogs(Object options)` function and pass in your message(s) as an option.
-* In your authentication journey/tree, add a scripted decision at a place where you want to output some content, and select the debugger script as the Script. Add "true" outcome to proceed with.
+* Using the example provided in the [src/scripted-debugger.js](src/scripted-debugger.js) file, call `showLogs(Object options)` function, and pass in your message(s) as the the `options.logs` argument.
```

Comment:
call the `showLogs(Object options)` function, and pass it in ... or do we mean paste it in ?

Reply:
Pass in, as in provide (an argument for the function).

Reply:
Thank you. Please ignore my comment.